en.json

```
@@ -417,7 +417,8 @@
                     "no_reason": "No ban reason."
                 },
                 "roles": {
-                    "title": "Roles"
+                    "title": "Roles",
+                    "unique_id": "This is current role unique ID."
```

IDs are static + minor wording changes:
```suggestion
                    "unique_id": "This is the role's unique ID."
```